Robert Underwood journaling article w/revisions

markcmiller86 opened the pull request:
# Description

EB Member: @markcmiller86 

Resolves #1572 

This PR replaces #1570. Try as I might, I was never able to submit changes to the original PR as stacked PR. So, I simply copied the article from it and submitted changes here.

## PR checklist for files displayed on bssw.io site

* [x] ***[Author]*** `@mention` the BSSw.io editorial board member `@<eb-member-id>` in **Description** above assigned to shepherd your PR.
* [x] ***[Author]*** Add the `<issue-id>` in the **Description** above for the associated GitHub Issue.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py])~~.
* [x] ***[Author]*** Inspect the content in the `*.md` file(s) as rendered in GitHub for this PR.
* [x] ***[EB Mem]*** Assign this PR to the EB member `<eb-member-id>`.
* [x] ***[EB Mem]*** Assign this PR to the author of the PR `<pr-author-id>`.
* [x] ***[EB Mem]*** Add one or more Reviewers.
* [x] ***[EB Mem]*** Add label `content: <content-type>` for the type of contribution.
* [x] ***[EB Mem]*** Add to Project `Content Development` (see [Content Development]).
* [x] ***[EB Mem]*** Add [meta-data] to the `*.md` file(s) (set `Publish: yes`).
* [x] ***[EB Mem]*** Add label `preview` (so PR branch will be merged to 'preview' branch and watch for possible merge failures).
* [x] ***[EB Mem]*** Rebuild [preview] site and confirm new content is there, renders correctly and is returned in searches.
* [x] ***[Author]*** Make any final changes to the PR based on feedback.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py]).~~
* [x] ***[EB Mem]*** Rebuild [preview] site and re-confirm content looks correct.
* [x] ***[EB Mem]*** Ensure at least one reviewer signs off on the final changes.
* [x] ***[EB Mem]*** Move the PR to "Ready to Publish" in [Content Development].
* [x] ***[EB Mem]*** Leave comment and @mention `@betterscientificsoftware/bssw-maint` (**BSSw Maint**) asking to carry out final publication steps.
* [x] ***[BSSw Maint]*** Verify that all needed files are present in the PR (article, images, updates to Site/Homepage.md carousel and/or Site/Announcements/Announcements.md as appropriate).
* [ ] ***[BSSw Maint]*** Merge PR. (Should automatically move to "Done" in [Content Development].)
* [ ] ***[BSSw Maint]*** Verify new contribution shows up on [bssw.io] as expected.

NOTE:
* Checklist items prefixed with ***[Author]*** are expected to be performed by the author of the PR or can be performed by the author.
* Checklist items prefixed with ***[EB Mem]*** must be performed by a [BSSw.io Editorial Board (EB) Member](https://betterscientificsoftware.github.io/bssw.io/bssw_members.html).
* Checklist items prefixed with ***[BSSw Maint]*** must be performed by someone in the `@betterscientificsoftware/bssw-maint` team (hint: type `@`, `b`, `s`, `s`, `w`, `-`, `m`  to auto-complete to `@betterscientificsoftware/bssw-maint`).

<!-- NOTE: Remove above checklist if using the below checklist for internal files. -->

<!-- NOTE: Remove below checklist if using the above checklist for  bssw.io files. -->


## PR checklist for (internal) files not displayed on bssw.io site

*Click "Write" above and remove comment markers to see below checklist items*

<!-- REMOVE THIS COMMENT MARKER IF USING BELOW CHECKLIST
* [ ] Set list of Reviewers (at least one).
* [ ] Add to Project [BSSw Internal].
* [ ] View the modified `*.md` files as rendered in GitHub.
* [ ] If changes are to the GitHub pages site under the `docs/` directory, consider viewing locally with Jekyll.
* [ ] Watch for PR check failures.
* [ ] Make any final changes to the PR based on feedback and review GitHub (and Jekyll) rendered files.
* [ ] Ensure at least one reviewer signs off on the changes.
* [ ] Once reviewer has approved and PR check pass, then merge the PR.
REMOVE THIS COMMENT MARKER IF USING ABOVE CHECKLIST -->


<!-- Standard links below, leave these this section! -->

[preview]: https://preview.bssw.io
[bssw.io]: https://bssw.io
[Content Development]: https://github.com/betterscientificsoftware/bssw.io/projects/3
[BSSw Internal]: https://github.com/betterscientificsoftware/bssw.io/projects/2
[meta-data]: https://betterscientificsoftware.github.io/bssw.io/bssw_styling_common.html#metadata-section
[wikize_refs.py]: https://github.com/betterscientificsoftware/bssw.io/blob/master/utils/README.md#wikize_refspy

markcmiller86 commented:
@ robertu94 please use this PR for article revision. You can view diffs in changes I propose [here](https://github.com/betterscientificsoftware/bssw.io/pull/1573/commits/9a88a92cdc59272dd1d6daf514660e6049c5a105)

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+
+### Productivity Journaling
+
+Productivity journaling is different from general journaling (e.g. keeping a diary).
```
```suggestion
Journaling for productivity is different what most people think of as keeping a diary.
```
I prefer the phrase `Journaling for productivity` to `productivity journaling`.  Productivity Journaling isn't some special kind of journaling which is wholly distinct from other kinds of journaling, but rather something which has some distinctive attributes when used for a purpose

I'm also not sure that all of the thinkers that I am interacting with in my writing would distinguish a diary from a journal.  I would agree that it's different than the "Dear diary, today I ..." motif that was popular in the 1800s and in childhood folk tales, but that's not how everyone uses that term even if that is the majority usage.

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+* **Tasks**: identify something that needs to be done (e.g. a bug to fix).
+* **Observations**: describe something that happened worth remembering almost like in a lab journal.
+* **Ideas**: capture unspecific but maybe important thoughts to be processed later (e.g. a better approach to a sorting problem).
+* **Lessons**: record key takeaways worth sharing with others (even your future self).
```
I like the addition of "even your future self here".

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+### Productivity Journaling
+
+Productivity journaling is different from general journaling (e.g. keeping a diary).
+Productivity journaling captures tasks, observations, ideas, and lessons to be periodically reviewed, distilled and migrated.
```
```suggestion
Journaling captures tasks, observations, ideas, and lessons to be periodically reviewed, distilled and migrated.
```

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+* **Ideas**: capture unspecific but maybe important thoughts to be processed later (e.g. a better approach to a sorting problem).
+* **Lessons**: record key takeaways worth sharing with others (even your future self).
+
+Effective productivity journaling also demands that one periodically slow down, pause, reflect and review their record of activity.
```
```suggestion
Effective  journaling also demands that one periodically slow down, pause, and reflect to review their record of activity.
```

small grammar thing: a of verbs has to have the same actor and object if used in a conjunctive expression.  One does not slow down their record of activity

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+
+Common reflection cadences are daily, weekly and monthly.
+Where appropriate, it might be worthwhile to align reflection cadence with a project's sprint cadence.
+[Building a Second Brain](https://www.buildingasecondbrain.com/) recommends *progressive summarization* which involves underlining important entries upon daily review, highlighting important underlined entries upon weekly review and migrating (copying) important highlighted entries upon monthly review.
```
This should start a separate paragraph because it is a distinct idea from the timeliness of periodic review.  Progressive summarization is used outside of periodic reviews. For example, you might use it in a literature review.

markcmiller86 replied:
resolved in #0b796631 (incoming)

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+At longer cadences, more substantial reflection and review is appropriate.
+For example, at monthly intervals, its reasonable to check in with yourself and ask...what should I stop, start, and continue for next month?
+Are my practices productive and effective or do I need to make some adjustments?
+At quarterly intervals, ask yourself...what is my vision/theme/aspiration(s) for this season?
```
I prefer the word seasonally to quarterly.  While season often means 4 month intervals, it can also mean a distinct period of time (i.e. college was a season of my life).  It is in this latter meaning that I meant season.

markcmiller86 replied:
Fixed in 4133751a8dbda094c1b88a07c103c992f307b18a. I seemed to have left a mix of `quarter` and `seasonal` in there in first revision so adjusted a bit more. Lemme know if better now.

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+For example, at monthly intervals, its reasonable to check in with yourself and ask...what should I stop, start, and continue for next month?
+Are my practices productive and effective or do I need to make some adjustments?
+At quarterly intervals, ask yourself...what is my vision/theme/aspiration(s) for this season?
+How well did I live up to my aspirations last quarter?
```
I think something is lost when we remove the idea of moving from one journal to the next.  Yes, copying paper journals isn't for everyone, but the idea here is to garbage collect your thoughts so that you have the most relevant and useful thoughts always close at hand.

markcmiller86 replied:
Added 1695f5d8a93d31927c924f860bbe72d9b5431ee0

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+
+#### Tools to consider
+
+There is a video series from [Tiago Forte](https://www.youtube.com/playlist?list=PLVNXAaej57W7fbYoc_XJ0bA0T3k_Ye-Q2) that describes a variety of journaling tools.
```
```suggestion
There is a video series from [Tiago Forte](https://www.youtube.com/playlist?list=PLVNXAaej57W7fbYoc_XJ0bA0T3k_Ye-Q2) that describes which journaling tools work best for different kinds of person.
```

i like breaking these tools out per section.  However the video describes more than just a variety of tools, but how to find a tool that will work best for you depending on your "note taking archetype", a concept that I thought was important, but just didn't quite make the cut in my original post.

markcmiller86 replied:
Fixed I think in a37507247a57d4368f35665f2cfc59e23a162612

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+
+When defining tasks, make them SMART -- specific, measurable, attainable, relevant, and timely.
+
+* Specific -- Specify as many of the [Five W's](https://en.wikipedia.org/wiki/Five_Ws) as possible (Who is responsible? What is the definition of done? When does it need to be completed? Where does it need to happen? Why is it necessary?)
```
The definition of done in my mind falls under "measurable", and some of the 5 w's also fall under other categories such as timely.  Additionally, some would push against including the 5 w's here because you don't need to have verbose entries for everything in your task manager for personal use that would answer these questions.  For example, I have an entry that just says `cusz tests` and I know what that means and who I need to talk to get it done. That is why I prefer the phrase unambigious here.

markcmiller86 replied:
Agreed...there is surely some overlap among the SMART categorizations. I think everyone would agree that a *deadline* is really part of being **Specific**. I don't some overlap really hurts though. I've changed `possible` to `practical`. That way, if some are not practical as in the `cusz test` example...thats fine.

Fixed in 8865e0378f317026612320bd09de46e71e9fbfc9

robertu94 replied:
I accept this with the rephrasing to "practical"

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+* Measurable -- Identify a quantifieable way for an observer to judge progress or completion. What are the artifacts or evidence of progress or completion?
+* Achievable -- Is everything needed (experience, skills, time, resources, etc.) to accomplish the task available? This may not always be clearly known before hand.
+* Relevant -- Is the task related to broader goals and themes?
+* Time-Bound -- What is the deadline? Or, what is the the maximum time you are willing to put into this task to move it forward?
```
I liked your description of time-bounded here.

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+
+#### Tools to consider
+
+If your work life is many short tasks that come and go quickly, consider a white board.
```
I vehemently disagree with this.  I use a whiteboard to manage very long running tasks.  I've also used pencil and paper for this.  I've even used a shared whiteboard in a software development team.  Yes, I will admit that a whiteboard doesn't scale to a point.  If you have a project the scope of ECP, you need something like Jira.  There are too many team members, that are too distributed, working on tens of thousands of tasks.  However, a whiteboard can scale to 30-80 tasks just fine.

Another piece of nuance that has been lost here is that you can use multiple tools for different things.  For example the whiteboard works well for projects, but doesn't do as well at reminding me that I need salsa from the grocery store -- I use Reminders for that.

markcmiller86 replied:
Not sure there is really enough here to be in *vehement* disagreement :wink:. 

Given the article's current length, I don't think all the nuance here needs to be captured.

Fixed in 7513fa8b69652c66c0b3b32120be2d9c01538e5f

robertu94 replied:
My objection was that the prior phrasing leaves the impression that whiteboards (and by implication pencil and paper, because almost anything you can do on a whiteboard you can do with paper), are "second class citizens" to digital tools relegated people with only short tasks to do.  As knowledge workers most of the tasks that we do are long.  Therefore I read this as whiteboards aren't useful for knowledge workers.  That's where my objection was coming from. *vehement* is probably a strong phrasing, but I really do think that 80+% of people or more should start with and as many as 50+% should stick with analogue tools.

I'm open to leaving the nuance about multiple tools out.  I just wanted to note it here.

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+When reviewing time logs, ask yourself if time could or should have been spent more productivly.
+Why did that happened?
+What could be automated/delegated?
+Don't be surprised if you wind up being surprised what you actually spend your time on.
```
```suggestion
You may be surprised what you actually spend your time on.
```
This was just awkward wording.

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+Automatic tools track application usage and, if configured properly, an account for the same application being used for different purposes.
+Some time tracking tools provide informative ways to visualize how time was spent as a summary.
+
+Consider [ActivityWatch](https://activitywatch.net/), [TimeWarrior](https://timewarrior.net/).
```
Activity watch is automatic, time warrior is a manual time tracking tool

markcmiller86 replied:
fixed in 0b3d8418f7b16f4e59df755364ba2746fbc94d5f

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+### Closing thoughts
+
+Specific tools are not as important as actual practice.
+Decent pencil and paper approaches can be adopted for every practice mentioned here.
```
```suggestion
Good pencil and paper approaches can be adopted for every practice mentioned here.
```

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,155 @@
+
+Specific tools are not as important as actual practice.
+Decent pencil and paper approaches can be adopted for every practice mentioned here.
+In fact, some schools of thought recommend using pencil and paper over other tools because its inexpensive, quick and nearly infinitely flexible.
```
```suggestion
In fact, some schools of thought recommend using pencil and paper over other tools because its inexpensive, quick and nearly infinitely flexible.
```

Everything that I've read on this subject recommends that people start with pencil and paper, establish good habits there, and then  switch to software solutions (with exceptions for time/location based reminders).  Yes there are a few purists that say digital tools are awful, but the vast majority of people in this space use digital tools for some aspect of their workflow.

markcmiller86 replied:
Nothing really requested here. But, did find a minor word change worth making based on your remarks...

fixed in 7a617116468bedafe75827751b705562abbeeba1

robertu94 reviewed (changes requested):
@markcmiller86 overall I agree with your changes.  In a few places I thought some nuance was lost or my meaning was changed.

robertu94 commented on Articles/Blog/2023-03-08-productivity-practices-and-tools.md:
```diff
@@ -0,0 +1,157 @@
+Are my practices productive and effective or do I need to make some adjustments?
+At longer intervals (e.g. seasonally), it is a good practice to ask yourself deeper questions...what is my vision/theme/aspiration(s) for this season?
+How well have I live up to my aspirations so far?
+At these more substantial reflection points, its often a good practice to bring the old journal to a close and start a *new* journal by copying key information (e.g. migrating) forward.
```
```suggestion
At these more substantial reflection points, its often a good practice to bring the old journal to a close and start a *new* journal by copying key information (e.g. migrating) forward to encourage reflecting on what is worth moving forward.
```

I think we need to add a clause that explains why we need to do migrate forward.  I'm open to different phrasings of this.

markcmiller86 commented:
@robertu94 thanks for quick review and feedback. I believe I have addressed all requests. Please take a look.

Also, sorry for not mentioning earlier. After making revisions such as these, I worry authors might not feel comfortable being the only author...especially if the revisions are not something they agree with (vehemently 😆). But, I also have no need to be included as author and find it better for bssw.io if I am not. So, In last commit, i have removed myself. I hope that is ok with you.

robertu94 commented:
@markcmiller86 I approve of the article in its current state.  I added what I hope is a clarifying comment on the "vehement" remark (and am starting to regret that particular word choice).  I feel that your edits have addressed my concerns from this morning.

I think you've done a nice job of cleaning up the article to fit with the style and tone of BSSW.  I think you should get recognition for your edits.  I'm open to ether "co-authorship", an  "edited by" byline, or an acknowledgement at the bottom of the article.    I leave the choice of these up to you.

markcmiller86 commented:
> I think you've done a nice job of cleaning up the article to fit with the style and tone of BSSW.

Thanks for this kind remark 😄 

> I think you should get recognition for your edits. I'm open to ether "co-authorship", an "edited by" byline, or an acknowledgement at the bottom of the article. I leave the choice of these up to you.

Ok, I will inquire with rest of editorial team on best appraoch.

@robertu94 thank you so much for writing this article. I suspect there will be another round of changes (certainly for grammar, spelling, etc....I dunno about you but I never seem to get commas right :wink:) and maybe a banner image to introduce it. But, we'll be back in touch with you about that soon. Thanks again.

markcmiller86 commented:
@rinkug and @bernhold this is ready for next round of review.

markcmiller86 commented:
FYI...once again the back end defeats *normal* markdown and html mixing. The side-by-side table, which appears fine on GitHub, doesn't render in preview on the site. I'll find another way to go which works with backend.

markcmiller86 commented:
93546793ac10c2c70e32601171acf5a61307c707 did not fix the table on the front end.

I think multiple issues could be occurring. Front may not like

* use of the pipe symbol (`|`) *within* a table cell (it is used in MD for delinating columns)
* use of `&nbsp;` in line immediately preceding table
* any line immediately preceding table
* table has too many columns (I think we might have a limit of 6 and this table has 7).

markcmiller86 commented:
Ok, trying to use pipe character as a character in a table cell is causing issue for back-end...

#### Tables as seen in GitHub markdown...

![Screen Shot 2023-03-14 at 12 37 02 PM](https://user-images.githubusercontent.com/5720676/225117847-20facb5d-1dbd-4487-9800-ca3c8d00688a.png)

#### Tables as rendered on site...

![Screen Shot 2023-03-14 at 12 36 41 PM](https://user-images.githubusercontent.com/5720676/225117922-b520fb25-4fa6-423e-ae51-6c253a15cd00.png)

I think best option is either to remove or to go with triple backtick.

rinkug commented:
> Ok, trying to use pipe character as a character in a table cell is causing issue for back-end...
> 
> #### Tables as seen in GitHub markdown...
![Screen Shot 2023-03-14 at 12 37 02 PM](https://user-images.githubusercontent.com/5720676/225117847-20facb5d-1dbd-4487-9800-ca3c8d00688a.png)
>

I am seeing the table fine on preview site. Although the md file of the article is missing the table headings. Is that intentional?

markcmiller86 commented:
> Although the md file of the article is missing the table headings. Is that intentional?

Thats because I gave up. I switched to back-ticked code code blocks. Ugly but works ok.

rinkug commented:
@robertu94 :  Please note the below points and pls respond. You can email us any preferred images.

Also, at the top of your article, we can include a "hero" image, which is also used as a background to highlight it as the newest article in the list of blog postings. This image is optional, but if you have ideas for an appropriate image please let us know. Many blog articles are published without hero images. 

Guidelines for "hero" image:
* wide rectangular shape; we can crop an image to the required size (1125 x 432 pixels)
* the design brief for our site focuses on three types of imagery for hero:
- stock images (we have access to a number of stock photo libraries),
- data visualizations (a.k.a. science images), or
- people in trainings, workshops, conferences, or other settings
* "hero" images should contain no bold text; incidental text (in diagrams, for instance) is OK.
 
We can share a science image from your own work or your facility based on the topic of your blog.  We work with a designer who can help adapt images that don’t quite fit the criteria.  Feel free to send a couple of options if you like (and let us know your own preferences among them).
  
* We request a brief bio for each of the authors
  - 50-100 words, in paragraph form, can include hyperlinks
  - Mention your current position, employer, a bit about your background
  - Include info about your interests related to software productivity and sustainability
  - Anything else you want to mention
  
* We use GitHub profile information (particularly photo and affiliations) to construct our Site Contributors page: <https://bssw.io/items/authors/>.  Please let us know your GitHub username that we should use. We encourage you to include a photo and up-to-date institutional affiliation in your GitHub profile.

* The article will be proof read by a professional editor, so it will still have updates.

rinkug commented:
> > Although the md file of the article is missing the table headings. Is that intentional?
> 
> Thats because I gave up. I switched to back-ticked code code blocks. Ugly but works ok.


Ok. I added table headings to your code blocks. Is it fine? https://preview.bssw.io/blog_posts/busy-productive

robertu94 commented:
@rinkug 

+ I don't have a scientific photo related to task management.  I considered something like a visualization of my task manager, but I thought that might be a bit too personal, or a bit too text heavy.  I found the following photo on unsplash which I think invokes thinking about tasks.  I've cropped it here for size and aspect ratio:

![tasks](https://user-images.githubusercontent.com/7604212/225168439-9c84b7c1-f6df-4bda-8341-f0c37fdebcb7.png)

https://unsplash.com/photos/cckf4TsHAuw

I'm open to better ideas if anyone has any 😄  choosing images isn't my area of expertise.

+ Bio

Robert Underwood is a Postdoctoral Appointee at Argonne National Laboratory in the Mathematics and Computer Science Division under Franck Cappello.  His research interests are at the [intersection of HPC, lossy compression, and data management](http://robertu94.github.io/about). He earned his Ph.D. in Computer Science from Clemson University in 2021.  He leads development of the [LibPressio data compression library ecosystem](http://robertu94.github.io/libpressio), is a contributor to the [SZ compression framework](http://szcompressor.org/), and participates in the [Joint Laboratory for Extreme Scale Computing](https://jlesc.github.io/).  Wherever he is, he advocates for software productivity through sustainable development. He writes articles as he is [learning to learn](http://robertu94.github.io/learning.html) on his website.

+ I've updated my github work affiliation and profile image.  My user name is @robertu94 

+ I'll await the edits by the copy-editor.

markcmiller86 commented:
@haikudeb I've made changes requested by Lois for headings and added bio. I think its ready for a grammar/syntax pass.

robertu94 commented:
@markcmiller86 I noticed a slight difference from the bio that I provided vs the one in the article.  The version I sent ends with

> He writes articles as he is [learning to learn](http://robertu94.github.io/learning.html) on his website

The version in the article says

> He writes articles on [his website](http://robertu94.github.io/learning.html) as he continues learning

Would you be okay if I change this back?  I use "learning to learn" as the "branding" for the set of articles that this one was based on.  I choose it because it emphasizes the the "meta" aspect of what I am trying to do with my articles.  I am trying to do more than teach time tracking, journaling, and task management.  I want to teach people to about the tradeoffs and the process of crafting a personal process for these tasks.

markcmiller86 commented:
> Would you be okay if I change this back? I use "learning to learn" as the "branding" for the set of articles

Sure thing. My apologies. I hadn't realized that intention. It read awkward to me.

rinkug reviewed (approved):
Looking good so far.

rinkug commented:
@robertu94 and @markcmiller86 : This is ready and will be published today.
